full-stack-tests/presentation/src/learning-snippets/Content.test.ts

```diff
@@ -11,582 +11,1002 @@ import { getFieldByLabel, tryGetFieldByLabel } from "../Utils";
+        const ruleset: Ruleset = {
+          id: "example",
+          rules: [{
+            ruleType: RuleTypes.Content,
+            specifications: [{
+              // load content for given input instances
+              specType: ContentSpecificationTypes.SelectedNodeInstances,
+            }],
+          }, {
+            ruleType: RuleTypes.ContentModifier,
+            class: { schemaName: "BisCore", className: "GeometricElement3d" },
+            propertyOverrides: [{
+              // force hide the UserLabel property
+              name: "UserLabel",
+              isDisplayed: false,
+            }, {
+              // force show the Parent property which is hidden by default through ECSchema
+              name: "Parent",
+              isDisplayed: true,
+              doNotHideOtherPropertiesOnDisplayOverride: true,
+            }, {
+              // override label of CodeValue property
+              name: "CodeValue",
+              labelOverride: "Overriden Label",
+            }],
+          }],
+        };
+        // __PUBLISH_EXTRACT_END__
+
+        // Ensure customizations have been made
+        const content = await Presentation.presentation.getContent({
+          imodel,
+          rulesetOrId: ruleset,
+          keys: new KeySet([{ className: "BisCore:Element", id: "0x61" }]),
+          descriptor: {},
+        });
+        expect(content!.descriptor.fields.length).to.eq(20);
+        expect(tryGetFieldByLabel(content!.descriptor.fields, "User Label")).to.be.undefined;
+        expect(tryGetFieldByLabel(content!.descriptor.fields, "Parent")).to.not.be.undefined;
+        expect(tryGetFieldByLabel(content!.descriptor.fields, "Overriden Label")).to.not.be.undefined;
+      });
+
     });
 
-    it("uses `calculatedProperties` attribute", async () => {
-      // __PUBLISH_EXTRACT_START__ ContentModifier.CalculatedProperties.Ruleset
-      // The ruleset has a content rule that returns content of given input instances. There's also
-      // a content modifier that creates a calculated property for `bis.GeometricElement3d` instances.
-      const ruleset: Ruleset = {
-        id: "example",
-        rules: [{
-          ruleType: RuleTypes.Content,
-          specifications: [{
-            // load content for given input instances
-            specType: ContentSpecificationTypes.SelectedNodeInstances,
+  });
+
+  describe("Specifications", () => {
+
+    describe("ContentInstancesOfSpecificClasses", () => {
+
+      let imodel: IModelConnection;
+
+      beforeEach(async () => {
+        await initialize();
+        imodel = await SnapshotConnection.openFile("assets/datasets/Properties_60InstancesWithUrl2.ibim");
+      });
+
+      afterEach(async () => {
+        await imodel.close();
+        await terminate();
+      });
+
+      it("uses `classes` attribute", async () => {
+        // __PUBLISH_EXTRACT_START__ ContentInstancesOfSpecificClasses.classes.Ruleset
```

Comment:
The snippet is created with `ContentInstancesOfSpecificClasses.classes.Ruleset` name, but inserted into docs with `ContentInstancesOfSpecificClasses.Classes.Ruleset`. That should be made consistent, and so far we've been using `PascalCase`.

This applies to all new snippets.